test/sql-luatest/gh_7063_fix_dequote_test.lua

```diff
@@ -0,0 +1,22 @@
+local server = require('test.luatest_helpers.server')
```

Comment:
Test name shouldn't be imperative, unlike commit titles. Here a more appropriate name would be `gh_7063_partial_dequote_test.lua` or just `gh_7063_dequote_test.lua`.

Reply:
Fixed.